net/net.sh

```diff
@@ -137,13 +143,19 @@ while [[ -n $1 ]]; do
     elif [[ $1 = --deploy-update ]]; then
       updatePlatforms="$updatePlatforms $2"
       shift 2
-    elif [[ $1 = --stake-internal-nodes ]]; then
+    elif [[ $1 = --stake-lamports-internal-nodes ]]; then
       stakeNodesInGenesisBlock="$2"
```

Comment:
Rename `stakeNodesInGenesisBlock` to `internalNodeStakeLamports`?   This no longer has anything to do with the genesis block